topo_processor/util/aws_files.py

```diff
@@ -62,3 +66,81 @@ def load_file_content(bucket_name: str, object_path: str) -> Dict[str, Any]:
 
 def build_s3_path(bucket_name: str, object_path: str) -> str:
     return f"s3://{bucket_name}/" + (object_path[1:] if object_path.startswith("/") else object_path)
+
+
+def create_s3_manifest(manifest_source_path: str) -> None:
+    # TODO:lock file
+    start_time = time_in_ms()
+    get_log().debug("check_manifest", manifestPath=manifest_source_path)
+
+    url_o = urlparse(manifest_source_path)
+    bucket_name = url_o.netloc
+    manifest_path = url_o.path[1:]
+    credentials: Credentials = get_credentials(bucket_name)
+
+    s3_client = boto3.client(
+        "s3",
+        aws_access_key_id=credentials.access_key,
+        aws_secret_access_key=credentials.secret_key,
+        aws_session_token=credentials.token,
+    )
+
+    try:
+        manifest_modified_datetime = s3_client.head_object(Bucket=bucket_name, Key=manifest_path)["LastModified"]
+        cutoff_datetime = datetime.now(timezone.utc) - timedelta(days=28)
```

Comment:
Why `days=28`?

Reply:
I have put a note in https://toitutewhenua.atlassian.net/browse/TDE-272 documenting the decision taken.